packages/visx-grid/src/grids/GridAngle.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import cx from 'classnames';
+import Line, { LineProps } from '@visx/shape/lib/shapes/Line';
+import { Group } from '@visx/group';
+import { ScaleInput, getTicks } from '@visx/scale';
+import { Point } from '@visx/point';
+
+import { CommonGridProps, GridScale } from '../types';
+import polarToCartesian from '../utils/polarToCartesian';
+
+export type GridAngleProps<Scale extends GridScale> = CommonGridProps & {
+  /** `@visx/scale` or `d3-scale` object used to convert value to position. */
+  scale: Scale;
+  /**
+   * Exact values used to generate grid lines using `scale`.
+   * Overrides `numTicks` if specified.
+   */
+  tickValues?: ScaleInput<Scale>[];
+  /**
+   * Radius which determines the start position of polar lines
+   */
+  innerRadius?: number;
+  /**
+   * Radius which determines the end position of polar lines
+   */
+  outerRadius?: number;
```

Comment:
I wonder if we should require `outerRadius`, I'm not sure of a use case where you'd want `innerRadius == outerRadius == 0`. Could potentially see utility in `innerRadius` defaulting to `0`, but the way it is now users would see nothing on the page if they didn't specify a value.

```suggestion
  outerRadius: number;
```